test/indexer/data/56a338fe-7554-4b5d-96a2-7df127a7640b.metadata.json

```diff
@@ -194,89 +194,114 @@
     },
     "project.json": {
         "content": {
-            "describedBy": "https://schema.humancellatlas.org/type/project/5.1.0/project",
+            "describedBy": "https://schema.humancellatlas.org/type/project/9.0.0/project",
             "project_core": {
                 "project_shortname": "Mouse Melanoma",
                 "project_description": "The cancer microenvironment is a complex ecosystem characterized by dynamic interactions between diverse cell types, including malignant, immune and stromal cells. Here, we performed single-cell RNA sequencing on CD45+ and CD45- cells isolated from tumour and lymph nodes during a mouse model of melanoma. The transcriptional profiles of these individual cells taken at different time points coupled with assembled T cell receptor sequences, allowed us to identify distinct immune subpopulations and delineate their developmental trajectory. Our study provides insights into the complex interplay among cells within the tumour microenvironment and presents a valuable resource for future translational applications.",
                 "project_title": "Melanoma infiltration of stromal and immune cells"
             },
-            "publications": [],
+            "publications": [
```

Comment:
Is this copied from a real bundle or did you write this from scratch?

Reply:
@hannes-ucsc I made this from scratch. I use the real metadata of a random medical research publication from NIH, which is unrelated to the bundle.